src/InsertionsClient/Models/PackageUpdateResult.cs

```diff
@@ -4,49 +4,49 @@
 
 namespace Microsoft.Net.Insertions.Models
 {
-	/// <summary>
-	/// Stores the result of a package version update operation
-	/// </summary>
-	public struct PackageUpdateResult : IEquatable<PackageUpdateResult>
-	{
-		/// <summary>
-		/// Id of the package that was updated.
-		/// </summary>
-		public string PackageId { get; private set; }
+    /// <summary>
+    /// Stores the result of a package version update operation
+    /// </summary>
+    public struct PackageUpdateResult : IEquatable<PackageUpdateResult>
+    {
+        /// <summary>
+        /// Id of the package that was updated.
+        /// </summary>
+        public string PackageId { get; private set; }
 
-		/// <summary>
-		/// Version of the package before the change.
-		/// </summary>
-		public string PreviousVersion { get; private set; }
+        /// <summary>
+        /// Version of the package before the change.
+        /// </summary>
+        public string PreviousVersion { get; private set; }
 
-		/// <summary>
-		/// Version of the package after the change.
-		/// </summary>
-		public string NewVersion { get; private set; }
+        /// <summary>
+        /// Version of the package after the change.
+        /// </summary>
+        public string NewVersion { get; private set; }
 
-		/// <summary>
-		/// Creates an instance of PackageUpdateResult.
-		/// </summary>
-		/// <param name="packageId">Id of the package that was updated.</param>
-		/// <param name="previousVersion">Version of the package before the change.</param>
-		/// <param name="newVersion">Version of the package after the change.</param>
-		public PackageUpdateResult(string packageId, string previousVersion, string newVersion)
-		{
-			PackageId = packageId;
-			PreviousVersion = previousVersion;
-			NewVersion = newVersion;
-		}
+        /// <summary>
+        /// Creates an instance of PackageUpdateResult.
+        /// </summary>
+        /// <param name="packageId">Id of the package that was updated.</param>
+        /// <param name="previousVersion">Version of the package before the change.</param>
+        /// <param name="newVersion">Version of the package after the change.</param>
+        public PackageUpdateResult(string packageId, string previousVersion, string newVersion)
+        {
+            PackageId = packageId;
```

Comment:
would validating these params add value?

Reply:
I don't believe so. We are storing the results of the default.config update operation here. Package wouldn't be updated if it wasn't valid.
I made the constructor internal so that users of the api cannot create invalid instances.